lib/routes.py

```diff
@@ -0,0 +1,64 @@
+from flask import request, Response, url_for
+from lib import app, db, socketio
+from flask_socketio import send, emit, join_room, leave_room, rooms
+from mongoengine import *
+from lib.models import Group
+
+
+
+# Test group room route
+@app.route('/groups', methods=['POST'])
+def add_group():
+  body = request.form
+  group = Group(**body).save()
+  return Response(group.to_json(), mimetype="application/json", status=200)
+
+# Send Message. Namespace is an example
+@socketio.on('message')
+def send_message(data):
+  room = data['room']
+  msg = data['message']
+  # print(data['message'], room=room)
+  send(msg, room=room)
+
+# Find a group
+def find_group(access_code):
+  Group.objects(access_code=access_code).get_or_404()
+
+@app.route('/groups', methods=['GET'])
+def get_groups():
+  groups = Group.objects().to_json()
+  return Response(groups, mimetype="application/json", status=200)
+  
```

Comment:
I have this method written in my local branch - I can add it in with the appropriate test for it

Reply:
Es bueno!